integration/performance_test.go

```diff
@@ -0,0 +1,76 @@
+package integration
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/CityOfZion/neo-go/config"
+	"github.com/CityOfZion/neo-go/pkg/core"
+	"github.com/CityOfZion/neo-go/pkg/core/storage"
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/network"
+	"github.com/stretchr/testify/require"
+)
+
+// Benchmark test to measure number of processed TX.
+// Same benchmark made on reference C# node https://github.com/neo-project/neo/issues/1321.
+func BenchmarkTXPerformanceTest(t *testing.B) {
+	net := config.ModeUnitTestNet
+	configPath := "../config"
+	cfg, err := config.Load(configPath, net)
+	require.NoError(t, err, "could not load config")
+
+	memoryStore := storage.NewMemoryStore()
+	chain, err := core.NewBlockchain(memoryStore, cfg.ProtocolConfiguration)
+	require.NoError(t, err, "could not create chain")
+
+	go chain.Run()
+
+	serverConfig := network.NewServerConfig(cfg)
+	server := network.NewServer(serverConfig, chain)
+	for n := 0; n < t.N; n++ {
+		tx := getTX()
+		require.Equal(t, server.RelayTxn(tx), network.RelaySucceed)
```

Comment:
Does `require.Equal` itself appear in benchmark results? May be it's better to separate test and benchmark?

Reply:
I don't think it would affect the results much, but since we do care about the result here for test correctness and at the same time it's a benchmark, not a functional test, maybe it'd be nice to not use `require` here with its magic, plain and simple `if server.RelayTxn(tx) != network.RelaySucceed` will do.

Reply:
done that way
btw results:
```
    5017            388022 ns/op          136509 B/op        459 allocs/op
PASS
ok      github.com/CityOfZion/neo-go/integration        2.610s
```